Source/sources/Castro_sources.cpp

```diff
@@ -61,10 +61,10 @@ Castro::source_flag(int src)
 #endif
 
     case geom_src:
-        if (geom.Coord() == 1) {
-          return true;
-        } else {
+        if (geom.Coord() == 0) {
```

Comment:
there aren't any geometry sources for 1-d spherical or cylindrical either, right?

Reply:
I think it depends on what does 1d spherical mean exactly. Are we assuming purely 1d with velocities in theta and phi direction being 0 or are we assuming 3d but with theta and phi symmetry?